2017-02/feb-22.md

```diff
@@ -11,9 +11,13 @@ Will do a few pre-releases to test; need to think about upgrade path for oss lib
 
 ### Getting Contributors
 
-- Big focus on [`beginner-friendly`](https://github.com/babel/babel/issues?q=is%3Aissue+label%3Abeginner-friendly+is%3Aclosed) issues for Babel 7: it's been really awesome to see the contributions
-- We are accepted to [Rails Girls Summer of Code](https://teams.railsgirlssummerofcode.org/projects/177-babel): please spread the word!
-- We're also in Google Summer of Code! https://summerofcode.withgoogle.com/organizations/5842528113786880/
+- Big focus on creating guided [`beginner-friendly`](https://github.com/babel/babel/issues?q=is%3Aissue+label%3Abeginner-friendly+is%3Aclosed) issues for Babel 7: it's been really awesome to see the majority of the changes coming from new contributors!
+
+[![Imgur](http://i.imgur.com/5Bndqfn.png)](https://github.com/babel/babel/issues/5184)
+
+- We are accepted to [Rails Girls Summer of Code](https://teams.railsgirlssummerofcode.org/projects/177-babel) as well as [Google Summer of Code](https://summerofcode.withgoogle.com/organizations/5842528113786880/)!
+- #summer-of-code slack channel
```

Comment:
Maybe add this to the previous bullet. For example:

>We are accepted to [Rails Girls Summer of Code](https://teams.railsgirlssummerofcode.org/projects/177-babel) as well as [Google Summer of Code](https://summerofcode.withgoogle.com/organizations/5842528113786880/)! Say "Hi" in the #summer-of-code Slack channel.

And make `#summer-of-code Slack channel` a link to our Slack.